scripts/comment-test-report.js

```diff
@@ -84,13 +84,20 @@ const parseReportJson = async ({ reportJsonUrl, fetch }) => {
                 } else {
                     arch = "unknown"
                 }
+                let lfcState = ""
+                if (test.parameters[1] == "True") {
```

Comment:
That looks pretty fragile, and at the same point, I don't see a clear way to get the LFC state from `suites.json` 🤔 

Should we change a boolean to a string (with LFC/without LFC) here
https://github.com/neondatabase/neon/blob/59ef70192591c0fc2a5c2a69eea0d47f7bd8e033/test_runner/fixtures/parametrize.py#L119

But need to keep a boolean value in a DB, so need to adjust this part as well:
https://github.com/neondatabase/neon/blob/59ef70192591c0fc2a5c2a69eea0d47f7bd8e033/scripts/ingest_regress_test_result-new-format.py#L137